### PR TITLE
Search button improvements

### LINKED ISSUE #666: Client widget article search should work on click of enter
Search button is not visible on careforever site.
https://careforever-for-acehelp.herokuapp.com/pages/about

### PR DESCRIPTION
**Pull request by chiraggshah**
Fixes #666

### CODE REVIEW

File: app/javascript/Client/Section/Search/SearchBar.elm [hunk 1]
```diff
@@ -44,14 +44,14 @@ type Msg
 
 view : Model -> String -> Html Msg
 view model color =
-    div [ class "ah-search-bar", style "background-color" color ]
+    Html.form [ class "ah-search-bar", style "background-color" color, onSubmit OnSearch ]
```
Comment (ritesh404) [inline app/javascript/Client/Section/Search/SearchBar.elm]: Why did we have to use a form here? `OnSearch` is anyway fired `onClick` of the span element
The issue here was to fix the styling. I think if you check the `search.scss` file you might find something that's breaking

**Comment by chiraggshah** [comment]
@ritesh404 : I thought the issue #666 had 2 requirements.
- Search button is not visible on careforever site. (this was as per the description of the issue)
I handled this in the first commit

- Client widget article search should work on click of enter (this was the title of the issue)
I thought what this meant was that the search should work on press of enter too. Thats why I implemented Form and onSubmit here in the second commit.

If the second point is not needed, then I can revert the commit. But I think it would be good to have.

**Comment by ritesh404** [comment]
@chiraggshah 

> Search button is not visible on careforever site. (this was as per the description of the issue)
I handled this in the first commit

Rather than inlining the style can you check why the styles specified we not working

> Client widget article search should work on click of enter (this was the title of the issue)
> I thought what this meant was that the search should work on press of enter too. Thats why I implemented Form and onSubmit here in the second commit.

Okay no worries. We can go with form submit

**Comment by chiraggshah** [comment]
@ritesh404 _a : Padding was used instead of margin. Fixed it.

**Pull request merged**